### PR TITLE
Capstone Project Setup

### PR DESCRIPTION
**Pull request by pushkar1995**
- Add basic requirements for repo setup 🔢 
- Add basic HTML and CSS according to the design 💯 
-  Add proper README file 👍 
- No Linters errors
-  Github Deployed Url : https://pushkar1995.github.io/capstone-project1.0/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+                        <a><i class="fa-brands fa-twitter"></i></a>
+                        <a>English</a>
+                        <a>Sign Up</a>
+                        <a>Login</a>
+                    </div>
+                </header>
+                <nav class="">
+                    <div class="desktop-menu">
+                        <div class="header-logo"><img src="JM-Images/JunglemanicLogo.png"></div>
+                        <div><a class="about-h4" href="/pages/about.html">About</a></div>
```
Comment (Shisui6) [inline index.html]: Nice job deploying your project using github pages. However, the link to your about page is not working as it leads to a 404 page. Please go through your project and ensure that all links work as expected

File: index.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        </div>
+    </section>
+
+    <section id='activities' class="main-activity-types-section">
+        <div class="activities-header-div">
+            <h3 class="activity-header">Main Activities @thejunglemanics</h3>
+        </div>
+        <div class="jm-activity-div">
+            <article class="jm-activity">
+                <div class="activity-icon"><img src="JM-Images/ActivitiesIcons/HikingIcon.png"></div>
```
Comment (Shisui6) [inline index.html]: Kindly make sure all the images in your website have an alt attribute to improve accessibility🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+                        <div class="section-two-container">
+                            <p>Join our community page as we explore Pokhara and its surroundings through hiking,
+                                camping, cycling, and motorcycle rides. We value nature, preserving it through
+                                sustainable
+                                practices. Experience thrilling adventures, connect with like-minded individuals, and
+                                create
+                                lasting memories in this vibrant and nature-centered community.
+                            </p>
+                        </div>
```
Comment (Shisui6) [inline index.html:79]: Currently your website is missing a lot of the styles from the behance design guidelines. Please go through all the points listed below and update your website to conform to the behance design:

Your design:
image

Behance design:
image


- Kindly update the main color of your website to be the same color used in the behance design. The main color used is `#fe5038`
- Currently there is a horizontal scroll in the desktop version of your website. Please identify the element causing it and remove it
- Kindly increase the font size and weight of the main title in the headline section  to match the behance design
-Kindly add some padding to each card in the mobile version of the main program section and align the text of each text to be in line with the behance design
- Kindly remove the activities and contact section. There are not present in the behance design
- Kindly make sure that each picture of your featured speakers are the same size
- Kindly add the partner section present in the behance design
- Kindly update the footer in your mobile version to have the logo aligned to the left and the text aligned to the right

File: pages/about.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+                        <a>Login</a>
+                    </div>
+                </header>
+                <nav class="">
+                    <div class="desktop-menu">
+                        <div class="header-logo"><a href='/index.html' class="header-logo"><img
+                                    src="/JM-Images/JunglemanicLogo.png"></a></div>
+                        <div><a class="about-h4" href="#top-section">About</a></div>
+                        <div>
+                            <h4><a href="#activities">Main Activities</a></h4>
```
Comment (Shisui6) [inline pages/about.html]: Currently, your about page does not represent the behance design at all. Please update your about page to include all the elements present in the behance design. Please make sure to respect the color scheme, typography and layout of each element

Your design:
image

Behance design:
image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @pushkar1995, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @pushkar1995 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**